Gemfile

```diff
@@ -25,5 +25,6 @@ group :development, :test do
 end
 
 group :deployment do
+  gem 'capistrano-bundler'
```

Comment:
capistrano is pulled in by capistrano-bundler. do you have a good example of the best practice for what to include here?

```
    capistrano-bundler (1.1.4)
      capistrano (~> 3.1)
      sshkit (~> 1.2)
```

common-accessioning does this:

```
  gem 'capistrano', '~> 3.0'
  gem 'capistrano-bundler', '~> 1.1'
  gem 'dlss-capistrano', '~> 3.1'
```